### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by JayaSamuthraDevi**

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+Hi, prioritize the work points
```
Comment (jothiswaranoh) [inline ReadMe.md:1]: no work now i will assign you

Comment (jothiswaranoh) [reply ReadMe.md:1]: index page iruka who right side images pannanum
our serive kojam improve panna pannum with background designs and box shadow podanum

Comment (JayaSamuthraDevi) [reply ReadMe.md:1]: okk

Comment (JayaSamuthraDevi) [reply ReadMe.md:1]: is the latest updates reflected on host . no styles.css and main.css changes where reflected which i made try to readd files and host it

Comment (JayaSamuthraDevi) [reply ReadMe.md:1]: i will start working by tomorrow tell me exactly what i need to

Comment (jothiswaranoh) [reply ReadMe.md:1]: ok

Comment (jothiswaranoh) [reply ReadMe.md:1]: hosting will happpen on tomorrow without any bugs

Comment (JayaSamuthraDevi) [reply ReadMe.md:1]: what about hosting work?

Comment (JayaSamuthraDevi) [reply ReadMe.md:1]: give a time like 5-6 or ur suitable time ,on that time lets come online

**Pull request closed without merging**